lidar_cluster_topic_name: /perception/livox_cluster

end_point_topic_name: /planning/end_point

allow_angle_error: 1.0 # degree

path_length: 82 # [m]
